themes/_default.yaml

```diff
@@ -23,8 +23,11 @@ generic:
     accent-darker: &accent-darker   '#0B4848'
     accent-dark: &accent-dark       '#177D7D'
     accent: &accent                 '#2A9595'
+    accent-border: &accent-border   '#E6BB00'
```

Comment:
http://paletton.com/palette.php?uid=73i0A0kmYv4gpRFk9G7qbq2qUf4 this was initial color system